README.md

```
@@ -2,18 +2,144 @@
 [![Codacy Badge](https://app.codacy.com/project/badge/Grade/dda6430161e146518704730d9916dba7)](https://www.codacy.com/gh/bosankus/Compose-Weatherify/dashboard?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=bosankus/Compose-Weatherify&amp;utm_campaign=Badge_Grade)
 [![Qodana](https://github.com/bosankus/Compose-Weatherify/actions/workflows/code_quality.yml/badge.svg)](https://github.com/bosankus/Compose-Weatherify/actions/workflows/code_quality.yml)
 
-## Weatherify
-### Using Jetpack Compose
+# Weatherify
+
+A modern weather application built with Jetpack Compose that provides current weather conditions, forecasts, and air quality information.
+
 [![Download APK](https://img.shields.io/badge/download-22272E.svg?style=for-the-badge&logo=android&logoColor=47954A)]( )
```

markdownlint-cli2 (0.17.2)

No empty links

(MD042, no-empty-links)